proto/cosmos/staking/v1beta1/genesis.proto

```diff
@@ -38,6 +39,93 @@ message GenesisState {
   repeated Redelegation redelegations = 7 [(gogoproto.nullable) = false];
 
   bool exported = 8;
+
+  // added newly added queues
+  repeated google.protobuf.Any buffered_msgs = 9;
+
+  // epoch number at the time of genesis
```

Comment:
Is this the current epoch we are in, or the next epoch to be executed?

Reply:
It's current epoch we are in